Quorum/apis/git_api/git_manager.py

```diff
@@ -73,9 +67,18 @@ def clone_or_update_for_repo(repo_name: str, repo_url: str, to_path: Path):
                 pp.pretty_print(f"Cloning {repo_name} from URL: {repo_url} to {repo_path}...", pp.Colors.INFO)
                 Repo.clone_from(repo_url, repo_path, multi_options=["--recurse-submodules"])
 
+
+    def clone_or_update(self) -> None:
+        """
+        Clone the repositories for the customer.
+
+        If the repository already exists locally, it will update the repository and its submodules.
+        Otherwise, it will clone the repository and initialize submodules.
+        """
+        
         for repo_name, repo_url in self.repos.items():
-           clone_or_update_for_repo(repo_name, repo_url, self.customer_modules_path)
+           GitManager.__clone_or_update_for_repo(repo_name, repo_url, self.modules_path)
```

Comment:
Why not self.__clone_or_update_for_repo(...)?

Reply:
When I'm using static methods I like to invoke them with the class name syntax. It helps me remind myself that these are static methods that are not related to the specific instance of the class.